lib/std/Build/Step/Run.zig

```diff
@@ -1391,6 +1391,19 @@ fn runCommand(
             }
         },
         else => {
+            // On failure, print stdout and stderr if captured.
+            const bad_exit = switch (result.term) {
+                .Exited => |code| code != 0,
+                .Signal, .Stopped, .Unknown => true,
+            };
+            if (bad_exit) {
+                if (result.stdio.stdout) |out| {
+                    try step.addError("stdout:\n{s}", .{out});
+                }
```

Comment:
stdout cannot be assumed to be printable. It may be a png image and fuck up your terminal for example.